types/koozaki__romaji-conv/index.d.ts

```diff
@@ -0,0 +1,66 @@
+// Type definitions for @koozaki/romaji-conv 2.0
+// Project: https://romaji-conv.koozaki.com/
+// Definitions by: Piotr Błażejewicz <https://github.com/peterblazejewicz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace romajiConv;
+
+/**
+ * RomajiConv のインスタンスを返す
+ *
+ * @param someString 変換対象の文字列
+ * @return のインスタンス
+ */
+declare function romajiConv(someString: string): romajiConv.RomajiConv;
+
+declare namespace romajiConv {
+    abstract class RomajiConv {
```

Comment:
I feel like I'm missing something, but why abstract?

Reply:
It's me, sometimes I'm so stick trying reasemble shape so stricly, I could forget about simplicity.
Amended, thx!